src/index.js

```diff
@@ -1,17 +1,17 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
 import './index.css';
-import App from './App';
-import reportWebVitals from './reportWebVitals';
+// import App from './App';
+import Calculator from './components/calculator';
+import 'bootstrap/dist/css/bootstrap.css';
 
 ReactDOM.render(
   <React.StrictMode>
-    <App />
+    <Calculator />
   </React.StrictMode>,
   document.getElementById('root'),
 );
 
 // If you want to start measuring performance in your app, pass a function
 // to log results (for example: reportWebVitals(console.log))
 // or send to an analytics endpoint. Learn more: https://bit.ly/CRA-vitals
```

Comment:
According to this requirement:

![image](https://user-images.githubusercontent.com/57421823/153051625-c18d0262-26f2-4dce-a0f1-95a120d5c388.png)

Therefore:

- [ ] Please, kindly remove all text, images, and styles added by CRA.
- [ ] OPTIONAL: Remove also the `App.test.js` created by the CRA.

You can find the CRA images in the `public` directory.
The styles added by CRA, are all included in `App.css`.

That way, the project will be free from useless files! 💪